content/glossary/artifact.md

```diff
@@ -16,17 +17,50 @@ Examples:
 - executable file
 - container image
 
-{{% notification type="info" %}}
 What all artifacts have in common is that they are all arrays of bytes ([]byte).
 
-**Two artifacts are equivalent IFF `[]byte(artifact1) == []byte(artifact2)`.**
-{{% /notification %}}
+## Artifact Equivalency
+
+Two artifacts are equivalent if and only if `[]byte(artifact1) == []byte(artifact2)`
+
+## Derived Artifacts
+
+Most artifacts are produced by a [build tool](/glossary/build_tool) consuming some set of input artifacts to produce an artifact as an output.
+Such artifacts are said to be 'derived artifacts'.
+
 
 ## Leaf Artifacts
 
-Some artifacts, usually source code files, are painstakingly constructed by hand by humans.  These are called 'leaf artifacts'.
+Artifacts which are not 'derived artifacts' as said to be 'leaf artifacts'.
+Leaf artifacts are usually source code files constructed by hand by humans.
 
 
-## Derived Artifacts
+Examples:
+
+- "foo.o is derived from foo.c and bar.h using gcc"
+- "fooexecutable is derived from foo.o and baz.o using ld"
+- "foo.class is derived from foo.java using javac"
+
+## Artifact Identifiers
+
+It should be possible to identify each artifact with an artifact id.
+
+Artifact IDs should have the following characteristics:
```

Comment:
They should, but many current methods don't. It would be instructive to pull in examples of artifact ids in the real world today, to give folks something to compare to, e.g. SWID tags. If desired, we could call out where they fall short wrt these characteristics.